### PR TITLE
Add options to avoid prompts

### PR DESCRIPTION
**Pull request by stevepiercy**
Source: https://github.com/collective/cookiecutter-plone-starter/?tab=readme-ov-file#use-options-to-avoid-prompts

### CODE REVIEW

**Comment by stevepiercy** [comment]
Trivial update to readme, can I skip the changelog? If not, I'll create one.

File: README.md [hunk 1]
```diff
@@ -70,6 +70,20 @@ pipx run cookieplone
 
 Cookieplone will walk you through the necessary steps, using sensible defaults and offering customization options where needed.
 
+### Use options to avoid prompts
+
+The cookiecutter will ask a lot of questions.
```
Comment (davisagli) [inline README.md:75]:
```suggestion
Cookieplone will ask a lot of questions.
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,20 @@ pipx run cookieplone
...
+You can use some of its options to avoid repeatedly entering the same values.
+
+#### `--no-input`
+
+You can use the [`--no-input`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-no-input) option to make the cookiecutter not prompt for parameters and only use `cookiecutter.json` file content.
```
Comment (davisagli) [inline README.md:80]:
```suggestion
You can use the [`--no-input`](https://cookiecutter.readthedocs.io/en/latest/cli_options.html#cmdoption-cookiecutter-no-input) option to make Cookieplone not prompt for parameters and only use `cookiecutter.json` file content.
```

**Review by davisagli — APPROVED** [review]

**Comment by stevepiercy** [comment]
@davisagli should I port these changes to https://github.com/collective/cookiecutter-plone-starter/?tab=readme-ov-file#use-options-to-avoid-prompts?

**Comment by davisagli** [comment]
@stevepiercy No, that already looks correct as written. With cookiecutter-plone-starter you ran cookiecutter directly instead of running Cookieplone as a wrapper of cookiecutter.

**Pull request merged**